transport/event_notifier.cc

```diff
@@ -268,6 +273,10 @@ void cql_server::event_notifier::on_leave_cluster(const gms::inet_address& endpo
 
 void cql_server::event_notifier::on_up(const gms::inet_address& endpoint)
 {
+    if (_endpoints_pending_joined_notification.erase(endpoint)) {
+        on_join_cluster(endpoint);
```

Comment:
Do we have a guarantee that on_up is always called and never missed? @asias may know

Reply:
It's supposed to be called, yes. But I can't claim that there are no bugs, of course.